docs/Neogeo-CD.md

```diff
@@ -0,0 +1,46 @@
+***
+_ The Neo Geo CD is the second home video game console of SNK Corporation's Neo Geo family, released on September 9, 1994, four years after its cartridge-based equivalent. The system was originally priced at US$399, or £399. It was discontinued in 1997 with sales of 570,000 systems._
```

Comment:
I don't care about the price, but if you want to mention it was cheapter than the original Neogeo AES, that's ok.